docs/getting_started.rst

```diff
@@ -1,20 +1,89 @@
+
+``EventsRouter`` processor in ``event_tracking_backends``, can route the transformed events (xAPI and Caliper format) to configured routers.
+
+The processors discussed above, are configured as nested backends (named ``xapi`` or ``caliper``) of `AsyncRoutingBackend`_ in ``EVENT_TRACKING_BACKENDS`` configuration of ``event-tracking`` library.
+
+Execution
+---------
+
+``RegexFilter`` and ``NameWhitelist`` run synchronously in the main thread. Processors in ``xapi`` and ``caliper`` backends are executed asynchronously, with each backend run as a separate celery task.
+
+Routing of transformed events is also done asynchronously. Therefore, nested celery tasks are created, one for each router, to route events that have been transformed by ``xapi`` or ``caliper`` backends.
+
+Retries
+-------
+
+Once an event fails to transmit due to connection error, it is retried after a delay of 30 seconds for 3 more times. If it still fails to transmit, then event is dropped unless it is configured to persist in the datebase.
+
+Persistence
+-----------
+
+Some events are critical for record keeping and should never be lost. These `business_critical_events`_ are persisted in the database even after the retries expire. Retry is attempted to transmit persisted events once a day. Following events are configured as ``business_critical_events``:
+
+#. ``edx.course.enrollment.activated``
+#. ``edx.course.enrollment.deactivated``
+#. ``edx.course.grade.passed.first_time``
+
+.. _business_critical_events: https://github.com/edx/event-routing-backends/blob/e375674156b347be833ad8c2479be2c4ff4b073f/event_routing_backends/helpers.py#L197
+
+Supported events
+----------------
+
+List of supported events can be found here :any:`supported-events`.
+
+
+Mapping of edX event onto xAPI and Caliper formats
+---------------------------------------------------
+
+Mapping for Caliper events can be found `Caliper mapping <https://docs.google.com/spreadsheets/d/1MgHddOO6G33sSpknvYi-aXuLiBmuKTfHmESsXpIiuU8/edit#gid=389163646>`_.
+
+Mapping for xAPI events can be found in `xAPI mapping <https://docs.google.com/spreadsheets/d/1hvOvJnWD9d00QjPoou0wTxx5gTsqk5uda6RJp56LJjI/edit?usp=sharing>`_.
+
+Version information of mapping
+------------------------------
+
+*Needs to be updated*
+
+Installation
+===============
+#. Upgrade edX devstack to latest master branch.
 
-Setup
-------------
-#. `event_tracking_backends` has two processors namely `CaliperProcessor` and `XApiProcessor` that can transform edX events into Caliper and xAPI format respectively. Events in Caliper format require an additional processor named `CaliperEnvelopeProcessor` before they can be routed.
+#. Start devstack and ssh into ``lms`` docker container (``make lms-shell``).
```

Comment:
I think these installation instructions should be generic instead of focusing just devstack.  So it should be

Install event routing backends library or add it to private requirements of your virtual environment(`requirements/private.txt`)

#. Run ``pip install edx-event-routing-backends``.
#. Run migrations (``python manage.py lms migrate``).
#. Restart LMS service of edx-platform